plugins/collaajUpdatePlugin/lib/model/collaajini.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+class collaajini
```

Comment:
This entire class looked to me complicated with no need, just hold the values in array and return them if they exists.
Also, I don't think it should be placed under model.